tests/machinery/test_base.py

```diff
@@ -820,6 +820,39 @@ def setUp(self):
             "filtered_retry_count_2": 0,
         }
 
+    def test_future_task_runner_sets_cid(self):
```

Comment:
Do we have a test covering when `cid` is not present?

Reply:
basically every existing test is checking that condition

Reply:
Sounds good. Thanks!